### ISSUE TITLE
No WCAG success criteria for users with Speech impairments

### ISSUE THREAD

**Issue by rahulpundir019** [w3c/wcag#4062]
### Resource URL

https://www.w3.org/WAI/WCAG22/Understanding/

### Add a description

Recently I was using a service where you can input your data only using your voice so when I checked WCAG guidelines, there is no Success criteria considering this as a failure use case so just want to know why there is no guidelines for these type of use cases.

**Comment by mraccess77** [comment]
It seems like a voice only interface which does not support a keyboard input interface would fail SC 2.1.1 Keyboard.

**Comment by rahulpundir019** [comment]
> It seems like a voice only interface which does not support a keyboard input interface would fail SC 2.1.1 Keyboard.

@mraccess77 Probably not if we take this use case, a keyboard user can activate the record button, which will pass the WCAG 2.1.1, but if that user has a speech impairment(Muteness), then the user will not be able to use that app. right?

btw, the app name was AirChat, which no longer exists. Read more at https://techcrunch.com/2024/04/13/airchat-launch/

**Comment by mraccess77** [comment]
> > It seems like a voice only interface which does not support a keyboard input interface would fail SC 2.1.1 Keyboard.
> 
> [@mraccess77](https://github.com/mraccess77) Probably not if we take this use case, a keyboard user can activate the record button, which will pass the WCAG 2.1.1, but if that user has a speech impairment(Muteness), then the user will not be able to use that app. right?
> 
> btw, the app name was AirChat, which no longer exists. Read more at https://techcrunch.com/2024/04/13/airchat-launch/

I was thinking of speech for navigation - but If the recording is for speech input via recording then a keyboard could be an alternative - e.g. type to Alexa or Siri and thus a lack of keyboard input for the recording feature could be a failure of SC 2.1.1.  But in other situations like recording a song which can't be put into text it seems like one of the exception might apply "except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints." - in this case perhaps the person vocal movements?

**Comment by patrickhlauke** [comment]
I'd suggest this is something to raise in http://github.com/w3c/wcag3/issues/

but fundamentally if a user is unable to operate a site or app using at least the keyboard (and from what is described here, they can only hit record, but NOT actually input data) then it will fail 2.1.1
